### PR TITLE
[DOCS] Add new resources and change sort order

### PR DESCRIPTION
**Pull request by ChrisChinchilla**
## Description

Adds some new solidity resources, sorts alphabetically, and removes a disclaimer than no longer seems relevant.

### Checklist
- [ ] Code compiles correctly
- [ ] All tests are passing
- [ ] New tests have been created which fail without the change (if possible)
- [ ] README / documentation was extended, if necessary
- [ ] Changelog entry (if change is visible to the user)
- [ ] Used meaningful commit messages

### CODE REVIEW

File: docs/resources.rst [hunk 1]
```diff
@@ -97,42 +97,48 @@ Discontinued:
 Solidity Tools
 ~~~~~~~~~~~~~~
 
+* `ABI to solidity interface converter <https://gist.github.com/chriseth/8f533d133fa0c15b0d6eaf3ec502c82b>`_
```
Comment (leonardoalt) [inline docs/resources.rst]:
```suggestion
* `ABI to Solidity interface converter <https://gist.github.com/chriseth/8f533d133fa0c15b0d6eaf3ec502c82b>`_
```

File: docs/resources.rst [hunk 2]
```diff
@@ -97,42 +97,48 @@ Discontinued:
...
-* `ABI to solidity interface converter <https://gist.github.com/chriseth/8f533d133fa0c15b0d6eaf3ec502c82b>`_
-    A script for generating contract interfaces from the ABI of a smart contract.
+* `EVM Lab <https://github.com/ethereum/evmlab/>`_
+    Rich tool package to interact with the EVM. Includes a VM, Etherchain API, and a trace-viewer with gas cost display.
+
+* `leafleth <https://github.com/clemlak/leafleth>`_
+    A documentation generator for Solidity smart-contracts.
+
+* `PIET <https://piet.slock.it/>`_
+    A tool to develop, audit and use solidity smart contracts through a simple graphical interface.
```
Comment (leonardoalt) [inline docs/resources.rst]:
```suggestion
    A tool to develop, audit and use Solidity smart contracts through a simple graphical interface.
```

**Comment by leonardoalt** [comment]
Needs to be rebased to fix the OSX build.

Also style checker is failing.

**Comment by stackenbotten** [comment]
There was an error when running `chk_coding_style` for commit `c43032fd30e26952341923ab95ddf560d0cf5d44`:
```
Error: Trailing whitespace found:
 docs/resources.rst:126: 

```
Please check that your changes are working as intended.

**Review by chriseth — APPROVED** [review]

**Pull request merged**